### DISCUSSION TITLE
[Unstable] Lying down with exosuit equipped possible

### DISCUSSION THREAD

**Discussion by Rehexr** [FakeFishGames/Barotrauma#13647] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Though normally you arent meant to be able to lie down while wearing an exosuit, it seems when putting on the exosuit after lying down, you can continue to be laid down while wearing one. 
![image](https://github.com/FakeFishGames/Barotrauma/assets/104232152/61cedbdd-dafc-464c-93e2-d94091d9b0e1)

### Reproduction steps

1.Drop exosuit by bed
2.Lie down
3.Equip exosuit
4.Congrats, you are now laid down wearing an exosuit

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Thank you for your report.

Tried to reproduce this bug with hospital bed and bunk bed, but it has been fixed.

Closing ticket.
